README.md

```diff
@@ -163,16 +164,16 @@ relics[0].describe()
 
 
 ### Config<a name="config"></a>
-A json text file named config located in ~/reliquery
+A json text file named config needs to be created in ~/reliquery 
```

Comment:
Reliquery doesn't require a config to use out of the box. It is set up to offer a default solution for storage. I think if we edit it to say something offers a the option to add a config. If you would like to customize your storage. something along those lines.